README.md

```diff
@@ -886,35 +886,59 @@ service. However, support is available to manually enable those services. To do
 following additional variables must be est on the instance:
 
 ```yaml
-DISCOVERY_ELASTICSEARCH_URL: "http://149.202.166.156:9200/"
 SANDBOX_ENABLE_DISCOVERY: yes
 SANDBOX_ENABLE_ECOMMERCE: yes
-DISCOVERY_VERSION: "c96c77bbf80650b3d3cc238335cc5205e24b775f"
+DISCOVERY_VERSION: 'open-release/ginkgo.1'
+nginx_discovery_gunicorn_hosts:
+    - "127.0.0.1"
+COMMON_HOSTNAME: "your-site-name"
+ECOMMERCE_PAYMENT_PROCESSOR_CONFIG:
+    ...
 ```
 
-Currently, we don't support Elasticsearch as a database backend, so we have to pass
-that in manually. Additionally, discovery and ecommerce still aren't first-class
-citizens, so they must be enabled via extra var. We need to set the hostname, other
-than the FQDN, that the appserver refers to itself as so that we can properly go out
-to the load balancer-terminated SSL connection. And finally, newer versions of course
-discovery don't work properly with OIDC login yet.
+We need to set the hostname to something other than the FQDN, that the appserver
+refers to itself as so that requests can properly through the load
```

Comment:
This was in the original, but if you want to change it…:
„as so that“ → „so that“
„that the appserver refers to itself“ ← I don't understand the „refers to itself“, maybe add punctuation?
„can properly through“ ← missing verb

Reply:
Thanks for spotting those typos -- I'd glossed over them!  Addressed by https://github.com/open-craft/opencraft/pull/294/commits/efab883dbcd784a4950ee15737be7a8b9e79e363#diff-04c6e90faac2675aa89e2176d2eec7d8R904

Reply:
Looks good, thanks.